bridge/slack/slack.go

```diff
@@ -265,6 +265,7 @@ func (b *Bslack) sendWebhook(msg config.Message) (string, error) {
 	return "", nil
 }
 
+// Note: returning an empty string skips the message
```

Comment:
Why? This is not true. It would rather mean that the message will not be registered, regardless of whether it was sent or not according to the code [here](https://github.com/42wim/matterbridge/blob/master/gateway/gateway.go#L330-L339).